sdk/tests/iot/provisioning/test_az_iot_provisioning_client_parser.c

```diff
@@ -640,6 +645,168 @@ static void test_az_iot_provisioning_client_no_logging_succeed()
   az_log_set_classification_filter_callback(NULL);
 }
 
+static void
+test_az_iot_provisioning_client_parse_received_topic_and_payload_json_custom_payload_succeed()
+{
+  az_iot_provisioning_client client = { 0 };
+  az_result ret = az_iot_provisioning_client_init(
+      &client, test_global_device_hostname, test_id_scope, test_registration_id, NULL);
+  assert_int_equal(AZ_OK, ret);
+
+  az_span received_topic = AZ_SPAN_FROM_STR("$dps/registrations/res/200/?$rid=1");
+  char received_payload[] = "{\"operationId\":\"" TEST_OPERATION_ID
```

Comment:
payload term is used in multiple places here. received_response?

Reply:
This is used throughout the tests: MQTT payload is indeed easily confused with DPS payload.
Will leave as-is.